doc/make/msys2.md

```diff
@@ -22,12 +22,44 @@ Here are the commands to install all dependencies needed to compile Lean on your
 pacman -S make mingw-w64-x86_64-cmake mingw-w64-x86_64-ccache mingw-w64-x86_64-gcc git
 ```
 
+Then make sure the following is included in your PATH:
+
+```bash
+export PATH=$PATH:/mingw64/bin
```

Comment:
This should definitely happen by default: https://github.com/msys2/MSYS2-packages/blob/85d54481612aea963af4ce822a6d9d98cb2a189b/filesystem/profile#L52
Is it clear under which circumstances it might not (which might lead to other issues)? Are you sure you started an "MSYS2 MinGW 64-bit" shell, not "MSYS2 MSYS"?

Reply:
Yes, that is probably not best to recommend as MINGW64 sets up other things too.

Reply:
Ah, you are right I was not using mingw64.exe.  My bad, I will fix it.